application/tests/build.xml

```diff
@@ -0,0 +1,30 @@
+<project name="testreport" default="phpunit" basedir=".">
+ <property name="package" value="MyApplication" override="true" />
+  <target name="clean">
+    <delete dir="build"/>
+  </target>
+  <target name="prepare">
+    <mkdir dir="build/logs"/>
+  </target>	
+ <target name="phpunit">
+ 	<phpunit printsummary="true" haltonfailure="false">
```

Comment:
@biechao I got the error ` PHPUnitTask requires PHPUnit to be installed`. How can I use my phpunit installed via Composer?

Reply:
phpunit should be run as global, I update the READEME file.
It should be placed as at /usr/local/bin/phpunit.